packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/detachLine/detach-line.controller.js

```diff
@@ -0,0 +1,76 @@
+export default class TelephonyLinePhoneOrderDetachCtrl {
+  /* @ngInject */
+  constructor($translate, OvhApiTelephony, TelephonyMediator, TucToast) {
+    this.$translate = $translate;
+    this.OvhApiTelephony = OvhApiTelephony;
+    this.TelephonyMediator = TelephonyMediator;
+    this.TucToast = TucToast;
+  }
+
+  $onInit() {
+    this.isLoading = true;
+    this.isDetached = false;
+    this.line = null;
+
+    return this.TelephonyMediator.getGroup(this.billingAccount)
+      .then((group) =>
+        group
+          .getLine(this.serviceName)
+          .getPhone()
+          .then((data) => {
+            this.phone = data;
+          }),
+      )
```

Comment:
**question:** No error handling?

Reply:
This is not necessary because if this function is in error you will not get to this section of the manager. Indeed this one is called previously to create the link to this section.